src/scripts/visualizations/tooltip.js

```diff
@@ -97,10 +98,21 @@
                     };
 
                     return clampPosition(pos);
+                },
+
+                'cursor': function cursorPositioner() {
+                    var pos = d3.mouse(layer[0][0]);
+                    var tooltip = tooltipElement[0][0]
+                    var offsetX = 40;
```

Comment:
My question was more about does this magic number work across all font-sizes/other stuff we don't know about or should we make this configurable